lib/resources/datav2/entityv2.ts

```diff
@@ -187,6 +187,86 @@ export interface RawLuld {
   z: string;
 }
 
+const cancel_error_mapping_v2 = {
+  S: "Symbol",
+	i: "ID",
+	x: "Exchange",
+	p: "Price",
+	s: "Size",
+	a: "CancelErrorAction",
+	z: "Tape",
+	t: "Timestamp",
+};
+
+export interface AlpacaCancelError {
+  Symbol: string;
+  ID: number;
+  Exchange: string;
+  Price: number;
+  Size: number;
+  CancelErrorAction: string;
+  Tape: string;
+  Timestamp: string;
+}
+
+export interface RawCancelError {
+  T: string;
+  S: string;
+  i: number;
+  x: string;
+  p: number;
+  s: number;
+  a: string;
+  z: string;
+  t: string;
+}
+
+const correction_mapping_v2 = {
+  S: "Symbol",
+	x: "Exchange",
+	oi: "OriginalID",
+	op: "OriginalPrice",
+	os: "OriginalSize",
+	oc: "OriginalConditions",
+	ci: "CorrectedID",
+	cp: "CorrectedPrice",
+	cs: "CorrectedSize",
+	cc: "CorrectedConditions",
+	z: "Tape",
+	t: "Timestamp",
```

Comment:
I think this is only bad formatted on the reviewing view, but please double check these.

Reply:
double-checked. Just github things :man_shrugging: